packages/toolpad-app/next.config.mjs

```diff
@@ -57,19 +57,39 @@ const regexEqual = (x, y) => {
   );
 };
 
+// See https://nextjs.org/docs/advanced-features/security-headers
+const securityHeaders = [
+  {
+    key: 'X-Frame-Options',
+    value: 'SAMEORIGIN',
+  },
```

Comment:
This might also create issues for self-hosters. I wouldn't be surprised if we get requests to remove this header. Now, it's probably safer to have it.

Reply:
agreed, we can add an option to whitelist /deploy/* over time of there's interest